cmd/oci-create-layer/oci-create-layer.1.md

```diff
@@ -0,0 +1,29 @@
+% OCI(1) OCI-CREATE-LAYER User Manuals
+% OCI Community
+% October 2016
+# NAME
+oci-create-layer \- Create filesystem changeset
+
+# SYNOPSIS
+**oci-create-layer** [child] [parent] [flags]
+
+# DESCRIPTION
+`oci-create-layer` creates a filesystem changeset from two layers. It compares child with parent and generates a filsystem diff, pack the diff into a uncompressed tar archive. The output tar archive name is the child name with .tar suffix by default, use `--dest` to specify a custom one.
```

Comment:
Can we write to stdout by default?  Then with #5 you could:

    $ oci-create-layer child-dir parent-dir | gzip | oci-cas put image-layout-dir

to push a layer into CAS without landing the uncompressed layer tarball on the disk.